pkg/config/vault.go

```diff
@@ -272,7 +272,7 @@ func populateCredentialsAsEnvs(config *StepConfig, secret map[string]string, key
 	vaultCredentialEnvPrefix, ok := config.Config["vaultCredentialEnvPrefix"].(string)
 	isCredentialEnvPrefixDefault := false
 
-	if !ok || len(vaultCredentialEnvPrefix) == 0 {
+	if !ok {
```

Comment:
if I understand correctly, for this change to be reflected correctly one has to set `vaultCredentialEnvPrefix = ""` . is this correct  ?